sql/src/main/java/io/crate/planner/ResultDescription.java

```diff
@@ -34,4 +38,43 @@
      * If it's empty it is on the handler node.
      */
     Collection<String> nodeIds();
+
+    /**
+     * OrderBy information a parent can use to do a sorted-merge.
+     * This needs to be set if (and only if) the result is pre-sorted.
+     *
+     * This DOES NOT cause the parent relation to do a FULL sort.
+     */
+    @Nullable PositionalOrderBy orderBy();
+
+    /**
+     * @return Limit that needs to be applied if the result is merged.
+     *         This does not indicate if a limit was applied to generate the result,
+     *
+     *         Therefore if a result was produced by applying the final limit it is allowed to return
+     *         {@link io.crate.operation.projectors.TopN#NO_LIMIT} to indicate that no additional limit needs to be applied.
```

Comment:
hm, how does the consumer of this result know if it should page or not if the description might say no_limit